### PR TITLE
Contactform

### PR DESCRIPTION
**Pull request by mj-e-boc**
-Add index.html file for the contact form 
-Add stylesheet for the contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,5 +72,238 @@ <h2 class="outro">LET'S CONNECT</h2>
...
+        required="required"
+      />
+      <textarea
+        name="textarea"
+        id="message"
+        cols="30"
+        rows="10"
+        placeholder="write your message herer"
+        maxlength="500"
+      ></textarea>
```
Comment (Bondok6) [inline index.html:305]: - [ ] Great job adding required attributes in email and text inputs 💯, but I suggest adding `required` attribute in the text-area too to activate the validation on submit. 👌

File: styles.css [hunk 2]
```diff
@@ -76,10 +76,245 @@ a {
...
+
+#submit {
+  width: 30%;
+  padding: 3%;
+  color: #6070ff;
+  border-radius: 10px;
+  border: 1px solid #cfd8dc;
+  margin-right: auto;
+  margin-left: 4.5%;
+}
```
Comment (Bondok6) [inline styles.css:320]: - [ ] According to the requirements you should implement the button interactions (enable, hover, etc.) based on your Figma design. so kindly implement them 👌

**Review by Bondok6 — CHANGES_REQUESTED** [review]
Hi @mj-e-boc and @jaamanu  🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- No Linter errors ✅
- Use Formspree correctly ✅
- Your repo is professional ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**